### ISSUE TITLE
--process_common_js_modules fails for rxjs

### ISSUE THREAD

**Issue by alexeagle** [google/closure-compiler#2424]
Angular depends on rxjs. So far, I've been building an ES2015-modules version of rxjs to make it work with Closure (doesn't matter if the language level is ES5 or ES2015, other than the latter optimizes slightly smaller). In https://github.com/ReactiveX/rxjs/issues/2335 I requested the rxjs team ship such a package for use with Closure.

However, IIUC closure should be able to consume the CommonJS/ES5 default distribution of rxjs. This would be nice since we could have a single dependency that gives the build-time nodejs runtime for Angular's compiler and also the sources to bundle into the app for the browser runtime.

Currently this does not work:
Clone https://github.com/alexeagle/closure-compiler-angular-bundling/tree/rxjs
(it includes
```
--js node_modules/rxjs/**.js
--process_common_js_modules
--module_resolution=node
```
It crashes on this file: https://unpkg.com/rxjs@5.3.0/symbol/iterator.js

```
$ yarn
$ yarn run build
```
=>
```

```java.lang.RuntimeException: java.lang.RuntimeException: INTERNAL COMPILER ERROR.
Please report this problem.

null
  Node(SCRIPT): node_modules/rxjs/symbol/iterator.js:1:0
"use strict";
  Parent: NULL
	at com.google.javascript.jscomp.CompilerExecutor.runInCompilerThread(CompilerExecutor.java:126)
	at com.google.javascript.jscomp.Compiler.runInCompilerThread(Compiler.java:752)
	at com.google.javascript.jscomp.Compiler.compile(Compiler.java:722)
	at com.google.javascript.jscomp.Compiler.compile(Compiler.java:692)
	at com.google.javascript.jscomp.AbstractCommandLineRunner.doRun(AbstractCommandLineRunner.java:1080)
	at com.google.javascript.jscomp.AbstractCommandLineRunner.run(AbstractCommandLineRunner.java:492)
	at com.google.javascript.jscomp.CommandLineRunner.main(CommandLineRunner.java:1890)
Caused by: java.lang.RuntimeException: INTERNAL COMPILER ERROR.
Please report this problem.

null
  Node(SCRIPT): node_modules/rxjs/symbol/iterator.js:1:0
"use strict";
  Parent: NULL
	at com.google.javascript.jscomp.ProcessCommonJSModules$RewriteModule.visitExport(ProcessCommonJSModules.java:824)
	at com.google.javascript.jscomp.ProcessCommonJSModules$RewriteModule.visit(ProcessCommonJSModules.java:685)
	at com.google.javascript.jscomp.NodeTraversal.traverseBranch(NodeTraversal.java:639)
	at com.google.javascript.jscomp.NodeTraversal.traverse(NodeTraversal.java:311)
	at com.google.javascript.jscomp.NodeTraversal.traverseEs6(NodeTraversal.java:582)
	at com.google.javascript.jscomp.ProcessCommonJSModules.process(ProcessCommonJSModules.java:129)
	at com.google.javascript.jscomp.Compiler.processAMDAndCommonJSModules(Compiler.java:1841)
	at com.google.javascript.jscomp.Compiler.parseInputs(Compiler.java:1549)
	at com.google.javascript.jscomp.Compiler.parse(Compiler.java:809)
	at com.google.javascript.jscomp.Compiler.compileInternal(Compiler.java:762)
	at com.google.javascript.jscomp.Compiler.access$000(Compiler.java:88)
	at com.google.javascript.jscomp.Compiler$2.call(Compiler.java:725)
	at com.google.javascript.jscomp.Compiler$2.call(Compiler.java:722)
	at com.google.javascript.jscomp.CompilerExecutor$2.call(CompilerExecutor.java:91)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
Caused by: java.lang.NullPointerException
	... 18 more

**Comment by ChadKillingsworth** [comment]
Looks like it doesn't like an exported computed name.

**Comment by alexeagle** [comment]
Awesome, I just tried again with today's release 20170409.0.0 and this is now working.

**Comment by ChadKillingsworth** [comment]
It was probably fixed by #2350 then
